nixos/modules/services/misc/jellyfin.nix

```diff
@@ -55,45 +58,6 @@ in
         StateDirectory = "jellyfin";
         CacheDirectory = "jellyfin";
         ExecStart = "${cfg.package}/bin/jellyfin --datadir '/var/lib/${StateDirectory}' --cachedir '/var/cache/${CacheDirectory}'";
```

Comment:
Shouldn't the `dataDir` be exposed like in other services? https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/misc/readarr.nix#L66

Reply:
I'm personally not a big fan of adding a `dataDir` option, as this makes it complicated to use `StateDirectory=`. I personally don't find the need to move `datadir` elsewhere. Do you have a usecase?

Reply:
I regularly use it to move data to a different mount as I don't like mixing folders and mounts inside `/var/lib`

Reply:
I'm new to this self hosting, but I was thinking of telling jellyfish to mount my external hd, like in /media/media-hd
Other tools like sonarr and radarr allow changing dataDir. I thought it would be nice at least for consistency. But again, maybe there's a proper approach to configure all these services and I'm missing it

Reply:
@woile note that Jellyfin's `--datadir` does not contain media, it contains runtime configuration (configured through the web interface), plugins, artist pictures, chapter screenshots, and other metadata.

Reply:
I've opened a [PR](https://github.com/NixOS/nixpkgs/pull/277220) to add this option and more before I was aware of this draft, hopefully nothing that'll get in the way here.